app/routes/video_routes.py

```diff
@@ -0,0 +1,86 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.helper import invalid_video, invalid_video_data
+from app.models.video import Video
+from app.models.rental import Rental
+
+video_bp = Blueprint("video_bp", __name__, url_prefix="/videos")
```

Comment:
The basic CRUD operations for videos are largely the same as for customers, so look for places where the feedback on customer endpoints could apply to video endpoints as well.